README.md

```diff
@@ -7,5 +7,8 @@
 
 [![Qodana](https://github.com/fungover/storm/actions/workflows/code_quality.yml/badge.svg)](https://github.com/fungover/storm/actions/workflows/code_quality.yml)
 
+![Coverage](.github/badges/jacoco.svg)
```

Comment:
This isn't following the same format as above, and when I checkout the branch it doesn't show a badge the way it's supposed to.

Reply:
Its because the badges executes when you push. Martin wanted it to be that way.